src/test/java/step3/domain/LottoTest.java

```diff
@@ -0,0 +1,44 @@
+package step3.domain;
+
+import static helper.Constants.*;
+import static org.assertj.core.api.Assertions.*;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+@DisplayName("로또 테스트")
+class LottoTest {
+
+    @ParameterizedTest(name = DISPLAY_NAME)
+    @CsvSource(value = "3,2,42,12,40,11")
+    void _6개의_숫자를_조합해_로또를_만들_수_있다(final int one, final int two, final int three,
+        final int four, final int five, final int six) {
+        //given
+        final List<LottoNumber> lottoNumbers = Arrays.asList(
+            new LottoNumber(one), new LottoNumber(two), new LottoNumber(three),
+            new LottoNumber(four), new LottoNumber(five), new LottoNumber(six)
+        );
+
+        //when
+        final Lotto lotto = new Lotto(lottoNumbers);
+
+        //then
+        assertThat(lotto).isNotNull().isEqualTo(new Lotto(lottoNumbers));
+    }
+
+    @ParameterizedTest(name = DISPLAY_NAME)
+    @CsvSource(value = "3,2,42,12,40,11,7")
+    void _6개의_숫자이상으로_조합해_로또를_만드려고_하면_예외가발생한다(final int one, final int two, final int three,
+        final int four, final int five, final int six, final int seven) {
```

Comment:
```suggestion
    @ParameterizedTest(name = DISPLAY_NAME)
    @CsvSource(value = {"3,2,42,12,40,11,7", "1,2,3,4,5,6,7"})
    void _6개의_숫자이상으로_조합해_로또를_만드려고_하면_예외가발생한다(final int one, final int two, final int three,
        final int four, final int five, final int six, final int seven) {
```

학습테스트로 진행했던 ParameterizedTest 활용 좋습니다 👍 
이 테스트의 장점을 사용하기 위해,  N개의 parameter를 넘겨주면 좋을 것 같아요 

파라미터를 지금처럼 하나만 쓰면 아래와 같이 표현해도 될 것 같아요

```
        //given
        final List<LottoNumber> lottoNumbers = Arrays.asList(
            new LottoNumber(3), new LottoNumber(2), new LottoNumber(42),
            new LottoNumber(12), new LottoNumber(40), new LottoNumber(11), new LottoNumber(7)
        );

        //when
        assertThatIllegalArgumentException().isThrownBy(() -> new Lotto(lottoNumbers));
```

Reply:
말씀대로, 하나의 케이스만 테스트하고있는 상황에서 `@ParameterizedTest`를 넣으니 가독성이 좋지 않고,
오히려 혼선만 주는 것 같다는 생각이 듭니다.

제안해주신 코드로 변경하도록 하겠습니다.